pkg/chaosdaemon/server_test.go

```diff
@@ -37,15 +37,15 @@ var _ = Describe("netem server", func() {
 	Context("newGRPCServer", func() {
 		It("should work", func() {
 			defer mock.With("MockContainerdClient", &MockClient{})()
-			_, err := newGRPCServer(containerRuntimeContainerd, &MockRegisterer{})
+			_, err := newGRPCServer(containerRuntimeContainerd, &MockRegisterer{}, tlsConfig{})
 			Expect(err).To(BeNil())
 		})
 
 		It("should panic", func() {
 			Ω(func() {
 				defer mock.With("MockContainerdClient", &MockClient{})()
 				defer mock.With("PanicOnMustRegister", "mock panic")()
-				newGRPCServer(containerRuntimeContainerd, &MockRegisterer{})
+				newGRPCServer(containerRuntimeContainerd, &MockRegisterer{}, tlsConfig{})
```

Comment:
Are there any tests on the new chaos-daemon grpc, with tls enabled? Or is it possible to introduce such tests?

Reply:
There isn't any test even without TLS :cry:. I will file an issue to add test for chaos-daemon server and client (with/without tls).

Reply:
#1536